docs/api/utility-process.md

```diff
@@ -1,26 +1,23 @@
-# UtilityProcess
+# utilityProcess
 
-`UtilityProcess` creates a child process with
+`utilityProcess` creates a child process with
 Node.js and Message ports enabled. It provides the equivalent of [`child_process.fork`][] API from Node.js
 but instead uses [Services API][] from Chromium to launch the child process.
 
-`UtilityProcess` is an [EventEmitter][event-emitter].
-
-## Class: UtilityProcess
-
-> Child process with Node.js integration spawned by Chromium.
-
 Process: [Main](../glossary.md#main-process)<br />
 
-### `new UtilityProcess(modulePath[, args][, options])`
+## Methods
+
+### `utilityProcess.fork(modulePath[, args][, options])`
 
-* `modulePath` string - Absolute path to the script that should run as entrypoint in the child process.
+* `modulePath` string - Path to the script that should run as entrypoint in the child process.
 * `args` string[] (optional) - List of string arguments that will be available as `process.argv`
   in the child process.
 * `options` Object (optional)
   * `env` Object (optional) - Environment key-value pairs. Default is `process.env`.
-  * `execArgv` string[] (optional) - List of string arguments passed to the executable. Default is `process.execArgv`.
-  * `stdio` (string[] | string) (optional) - Child's stdout and stderr configuration. Default is `pipe`.
+  * `execArgv` string[] (optional) - List of string arguments passed to the executable.
```

Comment:
Node.js defaults this to `process.execArgv`. What do we default it to?

Reply:
I don't think we can default to `process.execArgv`, there are two issues

1) We don't seem to populate `process.execArgv` in the main process, always returns `[]`
2) We only allow a limited set of CLI options to be propagated to Node for the main proces https://github.com/electron/electron/blob/7493062555b9eb4e6cc47d628a4831c121192f1f/shell/common/node_bindings.cc#L211-L215 so even if we fix 1) we cannot expand 2)

Can we leave the default for this option to be `[]` ?

Reply:
I'm fine for the default to be whatever, but we should document what it is, & what (kind of) options can be passed here.